scripts/setup-intellij.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+# Make sure ./bazel_configure.py is run before runing this scripts
+if [[ ! -f ./scripts/compile/env_exec.sh ]] ; then
+  echo "ERR: File ./scripts/compile_env/env_exec.sh is not found!"
```

Comment:
Thanks for the patch!

Please change `ERR` to `ERROR` and remove the exclamation point.

Also please accept the CLA, see http://twitter.github.io/heron/docs/contributors/community/